studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -0,0 +1,181 @@
+import { useAuthServiceStatusQuery } from 'data/service-status/auth-service-status-query'
+import { usePostgresServiceStatusQuery } from 'data/service-status/postgres-service-status-query'
+import { usePostgrestServiceStatusQuery } from 'data/service-status/postgrest-service-status-query'
+import { useStorageServiceStatusQuery } from 'data/service-status/storage-service-status-query'
+import { useSelectedProject } from 'hooks'
+
+const ServiceStatus = () => {
+  const { ref } = useParams()
+  const project = useSelectedProject()
+  const [open, setOpen] = useState(false)
+
+  const isBranch = project?.parentRef !== project?.ref
+  const { data } = useProjectApiQuery({ projectRef: ref })
+  const { endpoint, defaultApiKey } = data?.autoApiService || {}
+
+  // [Joshen] Realtime status will be handled separately
+  const [realtimeStatus, setRealtimeStatus] = useState<'loading' | 'success' | 'error'>('loading')
+  const realtimeClient =
+    endpoint && defaultApiKey ? createClient(`https://${endpoint}`, defaultApiKey) : undefined
+  if (realtimeClient !== undefined) {
+    const channel = realtimeClient.channel('health', { config: { broadcast: { self: true } } })
+    channel.on('broadcast', { event: 'health-check' }, (payload) => {
+      if (payload.event === 'health-check') {
+        setRealtimeStatus('success')
+        realtimeClient.removeChannel(channel)
+      }
+    })
+    channel.subscribe((status) => {
+      if (status !== 'SUBSCRIBED') return
+      channel.send({ type: 'broadcast', event: 'health-check', payload: {} })
+    })
+  }
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (realtimeStatus === 'loading') setRealtimeStatus('error')
+    }, 5000)
+  }, [])
+
+  // [Joshen] Still need pooler service check eventually
+  const {
+    isLoading: isLoadingPostgres,
+    isError: isErrorPostgres,
+    isSuccess: isSuccessPostgres,
+  } = usePostgresServiceStatusQuery({
+    projectRef: ref,
+    connectionString: project?.connectionString,
+  })
+  const {
+    isLoading: isLoadingPostgrest,
+    isError: isErrorPostgrest,
+    isSuccess: isSuccessPostgrest,
+  } = usePostgrestServiceStatusQuery({
+    projectRef: ref,
+  })
+  const {
+    isLoading: isLoadingStorage,
+    isError: isErrorStorage,
+    isSuccess: isSuccessStorage,
+  } = useStorageServiceStatusQuery({
+    projectRef: ref,
+  })
+  const {
+    isLoading: isLoadingAuth,
+    isError: isErrorAuth,
+    isSuccess: isSuccessAuth,
+  } = useAuthServiceStatusQuery(
+    {
+      projectRef: ref,
+      endpoint: endpoint,
+      anonKey: defaultApiKey,
+    },
+    { enabled: defaultApiKey !== undefined }
+  )
+
+  const services = [
+    {
+      name: 'Auth',
+      isLoading: isLoadingAuth,
+      isError: isErrorAuth,
+      isSuccess: isSuccessAuth,
+    },
+    {
+      name: 'Database',
+      isLoading: isLoadingPostgres,
+      isError: isErrorPostgres,
+      isSuccess: isSuccessPostgres,
+    },
+    {
+      name: 'Postgrest',
```

Comment:
IMO it's clearer this way:
```suggestion
      name: 'PostgREST',
```

Reply:
Alternatively just "REST" or "REST API"